### ISSUE TITLE
Release 2.0.0-M3 (motive: to add Scala 3.0.0-M2 support)

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-xml#471]
* [x] merge the 3.0.0-M2 PR (#470 by @isomarcte)
* [x] choose version number
  * there aren't any actual code changes since M2, but I'm going to call it M3 anyway since these are just milestones and it's a little bit easier this way 🤷‍♂️
* [x] create the release and write the notes
  * https://github.com/scala/scala-xml/releases/tag/v2.0.0-M3
* [x] wait for release build
  * https://travis-ci.com/github/scala/scala-xml/builds/204557524
* [x] release the staging repos
  * https://oss.sonatype.org/#stagingRepositories
* [x] artifacts on Maven Central
  * https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_3.0.0-M2/2.0.0-M3/
  * https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/2.0.0-M3/
  * https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.13/2.0.0-M3/
* [x] post-release, bump version in README
  *  45864edf31e1e6afa59df899cb2df296f135b7cf

**Comment by SethTisue** [comment]
weird, all the other artifacts have now shown up on Maven Central, but not https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.13/2.0.0-M3/

**Comment by ashawley** [comment]
There was a way to publish a tag with a suffix, e.g. `v2.0.0-M1#0.22.0-RC1`, not sure if that still works, but a new milestone probably is easier.

**Comment by ashawley** [comment]
There's nothing particularly surprising about the upgrade and I'm not sure how many users will be reading the changelog (do they ever?), but keep in mind that the changelog probably needs to represent the changes since 1.3.0, since that will more likely the upgrade path for users.

**Comment by SethTisue** [comment]
> There was a way to publish a tag with a suffix, e.g. v2.0.0-M1#0.22.0-RC1, not sure if that still works, but a new milestone probably is easier.

That special-tag stuff was removed by @lrytz; the new way to backpublish is to remove the unwanted matrix entries from `.travis.yml`, then tag and push the commit (tagged, but not on any branch). I did this recently over at https://github.com/scala/scala-parallel-collections/issues/143 and it worked fine. Not sure why I apparently had a fit of laziness here yesterday, it would have been only slight additional effort to keep the version number the same.

Agree that as the 2.0 series gets closer to final, there is more and more pressure to put proper release notes together.
